Library/Formula/wireshark.rb

```diff
@@ -3,13 +3,9 @@ class Wireshark < Formula
   homepage "https://www.wireshark.org"
 
   stable do
-    url "https://www.wireshark.org/download/src/all-versions/wireshark-1.12.8.tar.bz2"
-    mirror "https://1.eu.dl.wireshark.org/src/wireshark-1.12.8.tar.bz2"
-    sha256 "357e0a4e49525d80cdc740bb16539fcdb526ad38cc2ed6cabedafc9bdee5c7e7"
-
-    # Removes SDK checks that prevent the build from working on CLT-only systems
-    # Reported upstream: https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=9290
-    patch :DATA
+    url "https://www.wireshark.org/download/src/all-versions/wireshark-2.0.0.tar.bz2"
```

Comment:
@codeout Could you use `libpcap` resource as suggested for `Mavericks` and lower,

```diff
diff --git a/Library/Formula/wireshark.rb b/Library/Formula/wireshark.rb
index d58ad1f..b422e70 100644
--- a/Library/Formula/wireshark.rb
+++ b/Library/Formula/wireshark.rb
@@ -1,18 +1,9 @@
 class Wireshark < Formula
   desc "Graphical network analyzer and capture tool"
   homepage "https://www.wireshark.org"
-
-  stable do
-    url "https://www.wireshark.org/download/src/all-versions/wireshark-1.12.8.tar.bz2"
-    mirror "https://1.eu.dl.wireshark.org/src/wireshark-1.12.8.tar.bz2"
-    sha256 "357e0a4e49525d80cdc740bb16539fcdb526ad38cc2ed6cabedafc9bdee5c7e7"
-
-    # Removes SDK checks that prevent the build from working on CLT-only systems
-    # Reported upstream: https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=9290
-    patch :DATA
-
-    depends_on "homebrew/dupes/libpcap" => :optional
-  end
+  url "https://www.wireshark.org/download/src/all-versions/wireshark-2.0.0.tar.bz2"
+  mirror "https://1.eu.dl.wireshark.org/src/wireshark-2.0.0.tar.bz2"
+  sha256 ""
 
   bottle do
     sha256 "011b9c1f55fdb42223af49b7bec04f387d5dd89d5b8a600faecca2011b776f08" => :el_capitan
@@ -20,14 +11,6 @@ class Wireshark < Formula
     sha256 "7980680d82cb4c3bf01c6b8c1f3970767773d9a09d5573e3ba418fe08f224bb4" => :mavericks
   end
 
-  devel do
-    url "https://www.wireshark.org/download/src/all-versions/wireshark-2.0.0rc3.tar.bz2"
-    mirror "https://1.eu.dl.wireshark.org/src/wireshark-2.0.0rc3.tar.bz2"
-    sha256 "11f46b2d202f923c87a0b8e1a07bd9910bbcc5c265c69de3a23dde82e0f647e3"
-
-    depends_on "homebrew/dupes/libpcap" if MacOS.version == :mavericks
-  end
-
   head do
     url "https://code.wireshark.org/review/wireshark", :using => :git
 
@@ -61,7 +44,23 @@ class Wireshark < Formula
   depends_on "gtk+" => :optional
   depends_on "gnome-icon-theme" if build.with? "gtk+3"
 
+  resource "libpcap" do
+    url "http://www.tcpdump.org/release/libpcap-1.5.3.tar.gz"
+    sha256 "9ae92159c1060f15e6a90f2c4ad227268b6aaa382c316fa49a31c496b9979e93"
+  end
+
   def install
+    if MacOS.version <= :mavericks
+      resource("libpcap").stage do
+        system "./configure", "--prefix=#{prefix}/libexec/vendor",
+                              "--enable-ipv6",
+                              "--disable-universal"
+        system "make", "install"
+      end
+      ENV.prepend "CFLAGS", "-I#{prefix}/libexec/vendor/include"
+      ENV.prepend "LDFLAGS", "-L#{prefix}/libexec/vendor/lib"
+    end
+
     no_gui = build.without?("gtk+3") && build.without?("qt") && build.without?("gtk+") && build.without?("qt5")
 
     args = %W[
@@ -130,53 +129,3 @@ class Wireshark < Formula
     assert_equal "File name,Number of packets\ncapture.pcap,2\n", output
   end
 end
```